docs/nl/README.md

```diff
@@ -7,11 +7,38 @@ sidebarDepth: 2
 # Wiki :netherlands: <Badge text="LSSM Stable 3.3.5"/>
 
 ## Over de LSSM
+De LSS MANAGER V3 is een uitbreiding voor meldkamerspel.com
+
+Met deze extensie wordt een appstore aan de game toegevoegd, waarmee verschillende plug-ins kunnen worden geïnstalleerd. Alle functies zijn modulair opgebouwd - dus **u** kunt beslissen wat u activeert.
+
+Plug-ins die niet zijn geactiveerd, worden niet geladen - dit verhoogt de prestaties en het beheer.
 
 ## Installatie :inbox_tray:
+Door de LSSM te gebruiken, gaat u ermee akkoord dat we metadata verzamelen. U kunt [hier](/nl/metadata) meer informatie vinden over wat we  verzamelen.
 
 ### Eerste stap: Tampermonkey
+Als Tampermonkey nog niet in uw browser is geïnstalleerd, moet u dit doen. Hier is een overzicht van de links voor de meest voorkomende browsers:
+
+Browser|Link
+-------|----
+Chrome | [Download](https://chrome.google.com/webstore/detail/dhdgffkkebhmkfjojejmpbldmpobfkfo)
+Firefox| [Download](https://addons.mozilla.org/en-US/firefox/addon/tampermonkey/)
+Safari | [Download](https://safari.tampermonkey.net/tampermonkey.safariextz)
+Opera  | [Download](https://addons.opera.com/en/extensions/details/tampermonkey-beta/)
+
+Voor andere browsers u kan Tampermonky downloaden van [tampermonkey.net](https://www.tampermonkey.net/)
+
+::: Waarschuwing
+Houd er rekening mee dat we officieel geen ondersteuning bieden voor oudere browsers en mobiele browsers en Microsoft Edge (Internet Explorer). Ondersteuning voor deze browsers is niet gegarandeerd of waarschijnlijk.
+:::
 
 ### Tweede stap: Userscript
+Wanneer u Tampermonkey succesvol in uw browser heeft geïnstalleerd, kunt u [hier](https://github.com/LSS-Manager/lss-manager-v3/raw/master/lssm-v3.user.js) klikken of een nieuw Userscript maken met de volgende inhoud:
+
+<<< lssm-v3.user.js
+
+### Derde stap: Activeren
+
+Hierna volgt de indicator voor een actieve LSS-Manager:! [LSSM-Indikator] (/ img / lssm_navbar.png) Als je in meldkamerspel bent maar deze indicator niet in de rechterbovenhoek ziet, klik dan op het Tampermonkey-pictogram in je browser en controleer of de schakelaar voor LSS-Manager Script op "aan" staat.
```

Comment:
`! [LSSM-Indikator] (/ img / lssm_navbar.png)` must be`![LSSM-Indikator](/img/lssm_navbar.png)`

Reply:
https://github.com/LSS-Manager/docs/pull/12/commits/ad634de1c7b25525d5defefffc0eac5f51a3b734